pulsar/consumer_partition.go

```diff
@@ -1358,10 +1358,13 @@ func createEncryptionContext(msgMeta *pb.MessageMetadata) *EncryptionContext {
 	return &encCtx
 }
 
-func (pc *partitionConsumer) ConnectionClosed() {
+func (pc *partitionConsumer) ConnectionClosed(closeConsumer *pb.CommandCloseConsumer) {
 	// Trigger reconnection in the consumer goroutine
 	pc.log.Debug("connection closed and send to connectClosedCh")
-	pc.connectClosedCh <- connectionClosed{}
+	pc.connectClosedCh <- &connectionClosed{
```

Comment:
Can we pass one of them only by checking if tls is enabled or not here? I think we better pass one arg to the downstream funcs to simplify. (same for CommandCloseProducer)

Reply:
Done!